lib/bamboozled/api/base.rb

```diff
@@ -35,7 +35,11 @@ def request(method, path, options = {})
           case response.code
           when 200..201
             begin
-              JSON.parse(response).with_indifferent_access
+              if response.nil?
+                {"headers" => response.headers}.with_indifferent_access
```

Comment:
@kylefdoherty I believe this line will still throw an error. If `response` is nil, then `response.headers` will blow up. I think maybe you meant to check if `response.body.nil?`